modules/opp-architecture-installing-policyset.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="opp-architecture-installing-policyset_{context}"]
+= Installing {product-title} by using policy sets
+
+The {ocp} installation uses two policy sets to install additional products.
+
+[NOTE]
+====
+Edit the `policyGenerator.yaml` file to remove any products that you do not want to install. You can delete the product entries or comment out the lines.
+====
+
+.Procedure
+. Install the `PolicyGenerator` plug-in by following the instructions in _Installing and using the PolicyGenerator Kustomize plug-in_.
+. Clone the `policy-collection` repository:
++
+[source,terminal]
+----
+$ git clone https://github.com/stolostron/policy-collection
+----
+. Navigate to the policy set directory:
++
+[source,terminal]
+----
+$ cd policygenerator/policy-sets/stable/openshift-plus
+----
+. Generate and apply the policies by using the following command:
++
+[source,terminal]
+----
+$ kustomize build --enable-alpha-plugins | oc apply -f -
```

Comment:
@gparvin do we also want to cover the installation method that uses gitops/deploy.sh script?

Which is the preferred method `kustomize` or `deploy.sh script`?

Reply:
The deploy script is preferred because it uses the subscription controller that is a supported part of ACM.  Using it from github could cause users to pull in new features.  Looking now for a better procedure to reference in our doc.

Reply:
I added a comment that could be the gitops installation procedure which would go along with the doc in the ACM user's guide.